### PR TITLE
manifest: update SDC and MPSL

### PR DESCRIPTION
**Pull request by ibaz-nordic**
Update SDC and MPSL in nrfxlib.

Signed-off-by: Azizah Ibrahim <azizah.ibrahim@nordicsemi.no>

### CODE REVIEW

**Review by Tschet1 — CHANGES_REQUESTED** [review]
Please add a note in the release notes.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -244,14 +248,9 @@ nRF9160 samples
     * Minor logging and function structure improvements
     * :ref:`lib_nrf_cloud` library is no longer de-initialized and re-initialized on disconnect and reconnect.
 
-  * Added:
```
Comment (jori-nordic) [inline doc/nrf/releases/release-notes-changelog.rst:247]: why the removed lines ? I think you need to rebase.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -41,6 +41,14 @@ Protocols
 This section provides detailed lists of changes by :ref:`protocol <protocols>`.
 See `Samples`_ for lists of changes for the protocol-related samples.
 
+Bluetooth LE
+------------
+
+* Added support for changing the radio transmitter's default power level using :c:func:`sdc_default_tx_power_set`.
+* Added support for changing peripheral latency mode using :c:func:`sdc_hci_cmd_vs_peripheral_latency_mode_set`.
+
+For details, see `softdevice_controller_changelog`.
```
Comment (jori-nordic) [inline doc/nrf/releases/release-notes-changelog.rst:50]: Please fix the link:
```rst
For details, see :ref:`nrfxlib:softdevice_controller_changelog`.
```
I got this syntax from the previous release notes (nrf/doc/nrf/releases/release-notes-1.0.0.rst).

Comment (jori-nordic) [inline doc/nrf/releases/release-notes-changelog.rst:50]: Please fix the link:
```rst
For details, see :ref:`nrfxlib:softdevice_controller_changelog`.
```
I got this syntax from the previous release notes (nrf/doc/nrf/releases/release-notes-1.0.0.rst).

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (jori-nordic) [inline doc/nrf/releases/release-notes-changelog.rst:247]: why the removed lines ? I think you need to rebase.

**Review by jori-nordic — CHANGES_REQUESTED** [review]

**Comment by ibaz-nordic** [comment]
Retest results look good:
https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/latest%2Fsub%2Ftest-fw-nrfconnect-ble/detail/master/17223/pipeline
https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/latest%2Fsub%2Fdesktop52_verification/detail/master/12890/pipeline
https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/latest%2Fsub%2Ftest-sdk-find-my/detail/master/3915/pipeline

**Review by jori-nordic — APPROVED** [review]

**Review by Tschet1 — APPROVED** [review]

File: west.yml [hunk 3]
```diff
@@ -110,7 +110,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 766cc270caca3975fa6d924ada43177089040fcc
+      revision: pull/776/head
```
Comment (rlubos) [inline west.yml:113]: Please update SHA: `0fb04fb543c564430325b1df92c973a013632059`

**Comment by rlubos** [comment]
@greg-fer Any chance for a changelog review (the dedicated tech writer is OoO).

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -41,6 +41,14 @@ Protocols
 This section provides detailed lists of changes by :ref:`protocol <protocols>`.
 See `Samples`_ for lists of changes for the protocol-related samples.
 
+Bluetooth LE
+------------
+
+* Added support for changing the radio transmitter's default power level using :c:func:`sdc_default_tx_power_set`.
+* Added support for changing peripheral latency mode using :c:func:`sdc_hci_cmd_vs_peripheral_latency_mode_set`.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:48]:
```suggestion
* Added support for changing the radio transmitter's default power level using the :c:func:`sdc_default_tx_power_set` function.
* Added support for changing the peripheral latency mode using the :c:func:`sdc_hci_cmd_vs_peripheral_latency_mode_set` function.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Pull request merged**